api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -98,6 +102,10 @@ public void validate(Object target, Errors errors) {
 		
 		// check attributes
 		super.validateAttributes(visit, errors, Context.getVisitService().getAllVisitAttributeTypes());
+
+		validateStartDateTimeToBirthdate(visit, errors);
+
+		validateStartDateTimeToEstimatedBirthdate(visit, errors);
```

Comment:
If birthdate is not estimated, I'd think that you don't have to validate it again here since you've already validated it above